### PR TITLE
Add: Stepper Component

### LINKED ISSUE #55: Create Stepper component
Cf Figma

This should amount to extracting 

https://github.com/kiwicom/mobile/blob/master/packages/shared/src/buttons/IncrementDecrementButtons.js

### PR DESCRIPTION
**Pull request by RobinCsl**
Closes #55

### CODE REVIEW

File: src/Stepper/index.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  onDecrement,
+  number,
+  showNumber = true,
+  numberStyle,
+  min,
+  max,
+  disableDecrement: originalDisableDecrement,
+  disableIncrement: originalDisableIncrement,
+}: Props) {
+  const disableDecrement = (min && number <= min) || originalDisableDecrement;
```
Comment (lukewalczak) [inline src/Stepper/index.js]: `min && number <= min` and `max && number >= max` are used in some places. Maybe it's worth to extract it to const?

Comment (RobinCsl) [reply src/Stepper/index.js]: Good idea, I'll do that!

File: src/Stepper/index.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  onDecrement,
+  number,
+  showNumber = true,
+  numberStyle,
+  min,
+  max,
+  disableDecrement: originalDisableDecrement,
+  disableIncrement: originalDisableIncrement,
+}: Props) {
+  const belowMin = min && number <= min;
```
Comment (tbergq) [inline src/Stepper/index.js]: Would not `number <= min` be sufficient since
1 <= undefined -> `false`

File: src/Stepper/index.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  const aboveMax = max && number >= max;
+  const disableDecrement = belowMin || originalDisableDecrement;
+  const disableIncrement = aboveMax || originalDisableIncrement;
```
Comment (tbergq) [inline src/Stepper/index.js]: `||` -> `??`

File: src/Stepper/index.js (hunk 2 above)
Comment (RobinCsl) [reply src/Stepper/index.js]: I would get the following Flow error:
```
Cannot compare number [1] to undefined [2]. (References: [1] [2])
```

File: src/Stepper/StepperButton.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import Icon from '../Icon';
+import Touchable from '../Button/Touchable';
+import StyleSheet from '../PlatformStyleSheet';
+import type { StylePropType } from '../PlatformStyleSheet/StyleTypes';
+
+type ButtonProps = {|
+  +icon: React.Element<typeof Icon>,
+  +touchable: boolean,
```
Comment (ferrannp) [inline src/Stepper/StepperButton.js]: Does `touchable` need to be mandatory prop? Can we have it optional with default value?

File: src/Stepper/index.js [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+  max,
+  disableDecrement: originalDisableDecrement,
+  disableIncrement: originalDisableIncrement,
+}: Props) {
+  const belowMin = min && number <= min;
+  const aboveMax = max && number >= max;
+  const disableDecrement = belowMin ?? originalDisableDecrement;
+  const disableIncrement = aboveMax ?? originalDisableIncrement;
+
+  const commonIconProps = {
```
Comment (ferrannp) [inline src/Stepper/index.js]: Why common? It's not exported no?

File: src/Stepper/StepperButton.js (hunk 4 above)
Comment (RobinCsl) [reply src/Stepper/StepperButton.js]: Good point.

File: src/Stepper/index.js (hunk 5 above)
Comment (RobinCsl) [reply src/Stepper/index.js]: Renamed `commonIconProps` to `iconProps`. Thanks

**Pull request merged**